.github/ISSUE_TEMPLATE/3_product_release_checks.md

```diff
@@ -4,44 +4,111 @@ about: Use this template to track all release checks for your component
 title: "[Product] Release Checks"
 ---
 
-> [!IMPORTANT]
-> Follow the [guidance on how to use the templates](https://github.com/eclipse-tractusx/sig-release/blob/main/README.md#release-management-acceptance-criteria).
+<!-- 
+Thanks for your contribution! Please fill out this template as good as possible. 
+Important: Contributing Guidelines can be found here: https://eclipse-tractusx.github.io/docs/oss/how-to-contribute
+Checkout the repository README for process description. 
+-->
 
 ## Release Info
 
 Please provide information on what you want to be included in the Eclipse Tractus-X release.
 If you are not owner of this issue, please provide the information as comment to the issue.
 Make sure to assign this issue to expert(s) for their approval, as soon as you have finished preparation. Multiple assignees allowed; they will un-assign themselves once review completed.
 
-**Version to be included in Eclipse Tractus-X release**: *version placeholder*
+**Responsible contacts**:
+<!-- For this new release -->
+- Committer: *GitHub handle*
+- ExpertGroup: *GitHub handle*
+- Technical contact: *GitHub handle*
 
-**Leading product repository:** *repository link*
-
-# Compliance Verifications
+**Version to be included in Eclipse Tractus-X release**:
+<!-- Responsibility: Committer -->
 
-This issue tracks all checks, that need to be performed for a product release in Eclipse Tractus-X.
+- App version: *version placeholder*
+- Helm Chart version: *version placeholder*
 
-- [ ] **Interoperability** checks performed
-- [ ] **Data Sovereignty** checks performed
+**Leading product repository:** *repository link*
 
-# General Checks
+## Compliance
+<!-- Responsibility: Committer, ExpertGroup, Technical Committee for Standardization -->
 
+- [ ] Possible changes (through the new features) on related Catena-X standards are considered and addressed
 - [ ] [**Tractus-X Release Guidelines**](https://eclipse-tractusx.github.io/docs/release)(TRGs) fulfilled
 
 Make sure to open and fill in a separate **documentation issue** in your product repository using the [Quality Gate Checklist issue template](https://github.com/eclipse-tractusx/.github/blob/main/.github/ISSUE_TEMPLATE/qg-checklist.md)
 
 > [!NOTE]
 > Note: most criteria for documentation and security are now covered in TRGs
 
-# Security Checks
+## Functionality
+<!-- Responsibility: Committer, Testmanagement -->
+
+- [ ] Feature works as expected and described
+- [ ] Backward compatibility maintained (depending Major/ Minor) 
+
+## Performance
+<!-- Responsibility: Testmanagement -->
+
+- [ ] Memory and CPU usage is within acceptable limits oriented with goldilocks 
+
+## Testing
+<!-- Responsibility: Committer, Testmanagement -->
+
+- [ ] Unit tests cover all new features
+- [ ] Integration tests are updated
+- [ ] E2E/Integration test passed
+- [ ] Regressions tests passed
+- [ ] User Journey approved
```

Comment:
moving the point with the md file didn't answer my question: https://github.com/eclipse-tractusx/sig-release/pull/813#issuecomment-2309873532

Reply:
Hi Evelyn,
it seems this point slipped through on our side, and we apologize for the oversight. 
Since there are currently no user journeys in place, we have decided to remove this item. We expect that the new detailed test coverage will adequately address this area.